test_runner/fixtures/neon_fixtures.py

```diff
@@ -2724,6 +2751,47 @@ def get_metrics(self) -> SafekeeperMetrics:
         return metrics
 
 
+class S3Scrubber:
+    def __init__(self, env, remote_storage):
+        self.env = env
+        self.remote_storage = remote_storage
+
+    def scrubber_cli(self, args, timeout):
+        s3_storage = self.env.remote_storage
+        assert isinstance(s3_storage, S3Storage)
+
+        env = {
+            "REGION": s3_storage.bucket_region,
+            "BUCKET": s3_storage.bucket_name,
+        }
+        env.update(s3_storage.access_env_vars())
+
+        if s3_storage.endpoint is not None:
+            env.update({"AWS_ENDPOINT_URL": s3_storage.endpoint})
+
+        base_args = [self.env.neon_binpath / "s3_scrubber"]
+        args = base_args + args
+        r = subprocess.run(
+            args,
+            env=env,
+            check=False,
+            universal_newlines=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            timeout=timeout,
+        )
```

Comment:
This prompted me to look into the existing helpers.  I like capturing to files for the test report, but I also prefer to see errors in the log for quick debugging of short-lived commands like the scrubber, so I've enhanced `subprocess_capture` to enable tee-ing output to log and file, and used that here.

I anticipate in future we'll make the scrubber output a structured report so that we can allow-list certain anomalies in certain tests, so that output handler also knows how to capture  stdout.